qemu/tests/cfg/qsd_hotplug_vubp.cfg

```diff
@@ -0,0 +1,26 @@
+- qsd_hotplug_vubp:
+    type = qsd_hotplug_vubp
+    virt_test_type = qemu
+    start_vm = no
+    only qcow2 raw
+    only x86_64 i386
+    # QSD attributes
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1"
+    qsd_force_create_qsd1 = yes
+    # Image attributes
+    image_name_stg1 = images/stg1
+    image_size_stg1 = 11G
+    vm_mem_backend = memory-backend-memfd
+    vm_mem_share = yes
+
+    # Image export attributes
+    qsd_image_export_stg1 = {"type":"vhost-user-blk","num-queues":1}
+    drive_format_stg1 = vhost-user-blk-pci
+    image_vubp_props_stg1=  {"num-queues":1}
+    Linux:
+        guest_cmd = "dd if=/dev/urandom of=%s oflag=direct bs=1M count=1000"
+    Windows:
+        guest_cmd = "WIN_UTILS:\coreutils\DummyCMD.exe %s 1024000000 1"
+    ovmf:
+        no i440fx
```

Comment:
This seems meaningless.

Reply:
It avoid to ovmf work with i440fx

Reply:
since we only support x86 platform(the condition is in line 6), I think "only q35" is also ok.

Reply:
it may works on i440fx+seabios

Reply:
do you mean:
ovmf:
   only q35

Reply:
Yes, yours is also OK for me